### PR TITLE
@ #125 | should create docs on our content theme, category and keywords target

### LINKED ISSUE #125: should create docs on our content theme, category and keywords target
our long term, short term content keywords that should reveal about our brand and business.

We build this guide so that everyone can propose and create content following along.

### PR DESCRIPTION
**Pull request by hoatle**
connect #125 

This is the initial content from our wiki page with adjusted themes and topics.

We need everyone's contribution and suggestions to make it better.

### CODE REVIEW

**Comment by hoatle** [comment]
@hoavt please review and ask the team for more suggestions.

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -0,0 +1,210 @@
+# Contributing
+
+To contribute to this blog project.
+
+
+## Once upon a time...
+
+From day one, we set up company blog to share things along the Teracy journey. You can check the blog
```
Comment (hoavt) [inline CONTRIBUTING.md]: set up the company blog

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+posts at http://blog.teracy.com
+
+From day one, we needed a brand (logo, the color scheme for the brand, etc.) and we're still
+improving it. We did not have any revenue at that time but we cared and we did this from day one.
+
+From day one, we set up social media accounts (like twitter, facebook, etc)
```
Comment (hoavt) [inline CONTRIBUTING.md]: Should have a full stop at the end of the sentence

File: CONTRIBUTING.md [hunk 3]
```diff
@@ -0,0 +1,210 @@
...
+
+From day one, we set up public github organization account to create, work and collaborate on open
```
Comment (hoavt) [inline CONTRIBUTING.md]: set up a public ....

File: CONTRIBUTING.md [hunk 4]
```diff
@@ -0,0 +1,210 @@
...
+source projects.
+
+From day one, @hoatle talked a lot about #teracy on his personal account, on company facebook account
```
Comment (hoavt) [inline CONTRIBUTING.md]: on the company facebook...

**Comment by hoatle** [comment]
@hoavt feedbacks applied.

**Pull request merged**